tests/api.c

```diff
@@ -1143,6 +1143,43 @@ static void test_wolfSSL_CertManagerGetCerts(void)
           !defined(NO_FILESYSTEM) && !defined(NO_RSA) && \
           defined(WOLFSSL_SIGNER_DER_CERT) */
 }
+
+static int test_wolfSSL_CertManagerSetVerify(void)
+{
+    int ret = 0;
+#if !defined(NO_FILESYSTEM) && !defined(NO_CERTS) && \
+    !defined(NO_WOLFSSL_CM_VERIFY) && !defined(NO_RSA)
+    WOLFSSL_CERT_MANAGER* cm = NULL;
+    int tmp = myVerifyFail;
+    const char* ca_cert = "./certs/ca-cert.pem";
+    const char* ca_expired_cert = "./certs/test/expired/expired-ca.pem";
+
+    cm = wolfSSL_CertManagerNew();
+    AssertNotNull(cm);
+
+    /* Use the test CB that always accepts certs */
+    myVerifyFail = 0;
+    wolfSSL_CertManagerSetVerify(cm, myVerify);
+
+    ret = wolfSSL_CertManagerLoadCA(cm, ca_cert, NULL);
+    AssertIntEQ(ret, WOLFSSL_SUCCESS);
+
+    ret = wolfSSL_CertManagerVerify(cm, ca_expired_cert, WOLFSSL_FILETYPE_PEM);
```

Comment:
The `wolfSSL_CertManagerVerify` API should be used for non CA certificates (type = `CERT_TYPE`) (see ssl.c:6335). The test case should not be using a "CA".